src/my.js

```diff
@@ -11,7 +11,7 @@ var g = require('./constants');
 var debug = process.env.DEBUG ? true : false;
 var sysfsFiles = {};
 
-require = function (packageName, onfail) {
+myRequire = function (packageName, onfail) {
```

Comment:
Ah, glad this was caught.

Reply:
Can you squash the commits for fixes to bugs that you add? I highly agree with making the beautify commit separate so that the functional commit can be read easily, but distracting with added bugs is difficult.

Reply:
Sure, i will squash the commits into one , i did not know about this earlier , in future i will make sure to squash all the commits for fixes to bugs from my side.